docs/release-notes/NuGet-5.7.md

```diff
@@ -0,0 +1,94 @@
+
+* nuget.exe update -self -Source https://feed - [#1783](https://github.com/NuGet/Home/issues/1783)
+
+* Added support for multiple config files in %APPDATA%\NuGet directory - [#9394](https://github.com/NuGet/Home/issues/9394)
+
+* DeterministicSourcePaths now takes NuGet source packages into account - [#9431](https://github.com/NuGet/Home/issues/9431)
+
+* Added extern alias support for NuGet package references - [#4989](https://github.com/NuGet/Home/issues/4989)
+
+* Added INuGetProjectService.GetInstalledPackagesAsync extensibility API - [#9702](https://github.com/NuGet/Home/issues/9702)
+
+* Added interop API to enumerate fallback folders without requiring a solution/project - [#9395](https://github.com/NuGet/Home/issues/9395)
+
+* Added "latest" option for -MSBuildVersion - [#8808](https://github.com/NuGet/Home/issues/8808)
+
+### Issues fixed in this release
+
+**Bugs:**
+
+* Searching in PMUI with Static / Sleet package sources causes OutOfMemoryException - [#8478](https://github.com/NuGet/Home/issues/8478)
+
+* Improve DOTNET_HOST_PATH environment variable handling - [#7438](https://github.com/NuGet/Home/issues/7438)
+
+* nuget.exe spec Generates a copyright Tag with Hard-Coded Text of Copyright yyyy Instead of $copyright$ - [#8696](https://github.com/NuGet/Home/issues/8696)
+
+* nuget cli 3.5.0 throws exception 'authors required' during pack of a csproj ignoring placeholders and assemblyinfo attributes if the assembly name is changed - [#4234](https://github.com/NuGet/Home/issues/4234)
+
+* Consider updating the libraries to use newer versions of Newtonsoft.Json - [#9263](https://github.com/NuGet/Home/issues/9263)
+
+* HttpSourceAuthenticationHandler SemaphoreSlim future proofing - [#9463](https://github.com/NuGet/Home/issues/9463)
+
+* HttpRequestMessage gets reused multiple times which is not supported with the SocketHttpHandler - [#8661](https://github.com/NuGet/Home/issues/8661)
+
+* NuGet.Indexing 5.6.0 preview 3 and later use a different public key token - [#9481](https://github.com/NuGet/Home/issues/9481)
+
+* Honor TreatWarningsAsErrors during NuGet Package creation - [#7404](https://github.com/NuGet/Home/issues/7404)
+
+* [CPVM] Spurious package downgrades for multiple p2p projects  - [#9549](https://github.com/NuGet/Home/issues/9549)
+
+* The “Browse” tab is not aligned left with search box - [#9559](https://github.com/NuGet/Home/issues/9559)
+
+* The installed version is inconsistent with the embedded icon in the solution level PM UI for one package id with multiple versions installed - [#9321](https://github.com/NuGet/Home/issues/9321)
+
+* Leak: PartCreationPolicy(CreationPolicy.NonShared) NuGet.SolutionRestoreManager.RestoreOperationLogger - [#9595](https://github.com/NuGet/Home/issues/9595)
+
+* Avoid reading the assets file in no-op restores - [#9693](https://github.com/NuGet/Home/issues/9693)
+
+* Current Source Link package is not working - [#9695](https://github.com/NuGet/Home/issues/9695)
```

Comment:
@nkolev92 could you review this for wording? It's not really clear to me what it means.